src/dataframerow/dataframerow.jl

```diff
@@ -256,7 +257,13 @@ Base.get(f::Base.Callable, dfr::DataFrameRow, key::ColumnIndex) =
 Base.broadcastable(::DataFrameRow) =
     throw(ArgumentError("broadcasting over `DataFrameRow`s is reserved"))
 
-Base.NamedTuple(dfr::DataFrameRow) = NamedTuple{Tuple(keys(dfr))}(values(dfr))
+function Base.NamedTuple(dfr::DataFrameRow)
+    k = keys(dfr)
```

Comment:
`keys` will return a `Tuple` of `Symbol`s after #2199. So in general this PR should wait till we merge the other one as that one is much bigger

Reply:
In the end #2199 `keys` returns a `Vector`, but it makes a copy, so probably it is better to write:
```suggestion
    k = Tuple(_names(dfr))
```